# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../licq/licq-1.5.0-compile.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


error: invalid conversion from 'const char**' to 'char**'

--- licq-1.5.0/src/translator.cpp.orig	2011-04-19 02:00:23.472077820 +0200
+++ licq-1.5.0/src/translator.cpp	2011-04-19 02:09:10.076077820 +0200
@@ -394,7 +394,7 @@
   else
   {
     const char* inPtr = s.c_str();
-    size_t ret = iconv(tr, (ICONV_CONST char**)&inPtr, &inLen, &outPtr, &outLen);
+    size_t ret = iconv(tr, (char**)&inPtr, &inLen, &outPtr, &outLen);
     iconv_close(tr);
 
     if (outDone != NULL)
--- licq-1.5.0/plugins/osd/src/licq-osd.cpp.orig	2011-04-19 03:03:33.660077822 +0200
+++ licq-1.5.0/plugins/osd/src/licq-osd.cpp	2011-04-19 03:04:11.936077820 +0200
@@ -778,7 +778,7 @@
 
     while ((fromsize>0) && (tosize>0))
     {
-	if ((int)iconv(conv, (ICONV_CONST char **)&msgptr, &fromsize, &resptr, &tosize)==-1)
+	if ((int)iconv(conv, (char **)&msgptr, &fromsize, &resptr, &tosize)==-1)
 	{
 	    // array is not enough
 	    if (errno == E2BIG)
